app/services/api/api.ts

```diff
@@ -4,23 +4,20 @@ import { ApiConfig, DEFAULT_API_CONFIG } from "./api-config"
 import * as Types from "./api.types"
 import { QuestionSnapshot } from "../../models/question/question"
 import * as uuid from "react-native-uuid"
-import { decodeHTMLEntities } from "../../utils/html-decode"
 
-const API_PAGE_SIZE = 10
+const API_PAGE_SIZE = 50
 
 const convertQuestion = (raw: any): QuestionSnapshot => {
-  const id = uuid.v1().toString()
-  const decodedQuestion = decodeHTMLEntities(raw.question)
```

Comment:
Is this HTML decoding no longer needed?

Reply:
I will double check on this

Reply:
<img width="589" alt="CleanShot 2021-08-11 at 20 58 55@2x" src="https://user-images.githubusercontent.com/53795920/129224424-d9652138-a251-4865-ad39-00be44b77d0c.png">

This is from your blog and it seems decodedQuestion wasn't there already. However I found this in question-screen.tsx file. See the comment below:

```typescript
const renderQuestion = ({ item }) => {
  const question: Question = item
  return (
    <View style={QUESTION_WRAPPER}>
      {/* decoded here */}
      <Text style={QUESTION} text={decodeHTMLEntities(question.question)} /> 
      <RadioButtons
        options={question.allAnswers}
        onSelection={(guess: string) => onPressAnswer(question, guess)}
        selectedOption={question.guess}
        renderOption={renderAnswer}
      />
      <Button style={CHECK_ANSWER} onPress={() => checkAnswer(question)} text={"Check Answer!"} />
    </View>
  )
}
```